include/dsn/dist/replication/replication_types.h

```diff
@@ -5153,15 +5182,22 @@ class duplication_entry
     duplication_entry(duplication_entry &&);
     duplication_entry &operator=(const duplication_entry &);
     duplication_entry &operator=(duplication_entry &&);
-    duplication_entry() : dupid(0), status((duplication_status::type)0), remote(), create_ts(0) {}
+    duplication_entry()
+        : dupid(0),
+          status((duplication_status::type)0),
+          remote(),
+          create_ts(0),
+          fail_mode((duplication_fail_mode::type)0)
```

Comment:
I think use FAIL_SLOW is better than 0 here

Reply:
replication_types.h is auto-generated by thrift compiler. We do not change it.